content/docs/gto/user-guide/index.md

```diff
@@ -1,18 +1,12 @@
 # User Guide
 
-GTO helps you build an Artifact Registry out of your Git repository. It creates
-annotated [Git tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging) with a
-[special format](#git-tags-format), and manages an `artifacts.yaml` file.
+GTO and DVC help you build an Artifact Registry out of your Git repository. GTO
+creates annotated [Git tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging)
+with a [special format](#git-tags-format), while DVC manages an
+[`dvc.yaml`](https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#artifacts)
+file with metainformation about the artifact and stores artifact binaries.
 
-<admon type="tip">
-
-Storing large files in Git repos is not a good practice. Avoid committing your
-ML artifacts to Git. You can [use DVC](/doc/gto/user-guide/dvc), Git LFS, or any
-other method to commit pointers to the data, models, etc. instead.
-
-</admon>
-
-## Annotations in artifacts.yaml
+## Annotations in dvc.yaml
```

Comment:
Feels odd to talk about dvc.yaml as the first section heading in the GTO guide. Is it needed? Can it be further down, and can we clarify better that it's an optional integration? Unlike for something like the Studio docs, this is the place to explain specifically what GTO does.